content/syllabuses/teaching.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Teaching
+---
+
+- {{% contentlink path="teaching_curriculum/introducing_teach_like_a_champion" %}}
```

Comment:
looks like you changed the folder naming from snake_case back to kebab-case(I think that's what its called). Not anything major as we seem to use both, but keep in mind all content links would need to be updated as well. 

I will make this change as the linter will help me check that the changes are correct.